### ISSUE TITLE
(A doubt NOT an issue) Gradient of loss function having different sign after performing first-degree differentiation of LogisticLoss

### ISSUE THREAD

**Issue by mehulofficial14** [eriklindernoren/ML-From-Scratch#107]
I differentiated the loss function in the LogisiticLoss() class used in XGboost to get the gradient. I am getting (y - p) but turns out that it is -(y - p). For more reference, here is the [link](https://github.com/eriklindernoren/ML-From-Scratch/blob/master/mlfromscratch/supervised_learning/xgboost.py) to the exact class and function, I am referring to. Am I missing something?
